docs/metrics.md

```diff
@@ -54,14 +54,37 @@ The following table describes the available metrics.
+|              |Failure Count   |Number of failed inference requests received by Triton (each request is counted as 1, even if the request contains a batch) |Per model  |Per request  |
 |              |Inference Count |Number of inferences performed (a batch of "n" is counted as "n" inferences)|Per model|Per request|
 |              |Execution Count |Number of inference batch executions (see [Count Metrics](#count-metrics))|Per model|Per request|
-|Latency       |Request Time    |Cumulative end-to-end inference request handling time    |Per model  |Per request  |
-|              |Queue Time      |Cumulative time requests spend waiting in the scheduling queue     |Per model  |Per request  |
-|              |Compute Input Time|Cumulative time requests spend processing inference inputs (in the framework backend)     |Per model  |Per request  |
-|              |Compute Time    |Cumulative time requests spend executing the inference model (in the framework backend)     |Per model  |Per request  |
-|              |Compute Output Time|Cumulative time requests spend processing inference outputs (in the framework backend)     |Per model  |Per request  |
+|Latency       |Request Time    |Cumulative end-to-end inference request handling time (includes cached requests) |Per model  |Per request  |
+|              |Queue Time      |Cumulative time requests spend waiting in the scheduling queue (includes cached requests) |Per model  |Per request  |
+|              |Compute Input Time|Cumulative time requests spend processing inference inputs (in the framework backend, does not include cached requests)     |Per model  |Per request  |
+|              |Compute Time    |Cumulative time requests spend executing the inference model (in the framework backend, does not include cached requests)     |Per model  |Per request  |
+|              |Compute Output Time|Cumulative time requests spend processing inference outputs (in the framework backend, does not include cached requests)     |Per model  |Per request  |
+|Response Cache|Total Cache Entry Count |Total number of responses stored in response cache across all models |Server-wide |Per second |
+|              |Total Cache Lookup Count |Total number of response cache lookups done by Triton across all models |Server-wide |Per second |
+|              |Total Cache Hit Count |Total number of response cache hits across all models |Server-wide |Per second |
+|              |Total Cache Miss Count |Total number of response cache misses across all models |Server-wide |Per second |
+|              |Total Cache Eviction Count |Total number of response cache evictions across all models |Server-wide |Per second |
+|              |Total Cache Lookup Time |Cumulative time requests spend checking for a cached response across all models (microseconds) |Server-wide |Per second |
+|              |Total Cache Utilization |Response Cache utilization rate (0.0 - 1.0) |Server-wide |Per second |
```

Comment:
Doesn't utilization of 0.0-100.0 make more sense?

Reply:
I wanted to keep it consistent with the existing `GPU Utilization rate (0.0 - 1.0)` metric